core/jobs/batch_jobs/exp_validation_jobs.py

```diff
@@ -0,0 +1,828 @@
+            rte_components = (html_cleaner.get_rte_components(
+                value.content.html))
+            if len(rte_components) > 0:
+                for rte_component in rte_components:
+                    if rte_component['id'] == 'oppia-noninteractive-image':
+                        if len(rte_component['customization_args']
+                        ['caption-with-value']) > 160:
+                            rte_components_errors.append(
+                                'State - ' + str(key) +
+                                ' Image tag caption value'
+                                + ' is greater than 160.')
+                        if len(rte_component['customization_args']
+                        ['alt-with-value']) < 5:
+                            rte_components_errors.append(
+                                'State - ' + str(key) +
+                                ' Image tag alt value' + ' is less than 5.')
+                        if (rte_component['customization_args']
+                        ['filepath-with-value'][-3:] != 'svg'):
+                            rte_components_errors.append(
+                                'State - ' + str(key) +
+                                ' Image tag filepath value' +
+                                ' does not have svg extension.')
+
+                    elif rte_component['id'] == 'oppia-noninteractive-math':
+                        svg_filename = (rte_component['customization_args'][
+                            'math_content-with-value']['svg_filename'])
+                        if svg_filename[-3:] != 'svg':
+                            rte_components_errors.append(
+                                'State - ' + str(key) +
+                                ' Math tag svg_filename value'
+                                + ' has a non svg extension.')
+
+                    elif (rte_component['id'] ==
+                        'oppia-noninteractive-skillreview'):
+                        if (rte_component['customization_args']
+                        ['text-with-value']) == '':
+                            rte_components_errors.append(
+                                'State - ' + str(key) +
+                                ' Skill review tag text value' + ' is empty.')
+
+                    elif rte_component['id'] == 'oppia-noninteractive-video':
+                        start_value = (
+                            rte_component['customization_args']
+                            ['start-with-value'])
+                        end_value = (
+                            rte_component['customization_args']
+                            ['end-with-value'])
+
+                        if int(start_value) > int(end_value):
+                            rte_components_errors.append(
+                                'State - ' + str(key) +
+                                ' Video tag start value' + ' is greater than'
+                                ' end value.')
```

Comment:
In multiline strings put space at the end of the lines. Also don't use `+` for concatenating strings, use string formatting

Reply:
Done